packages/neuron-ui/.eslintrc.js

```diff
@@ -26,9 +26,7 @@ module.exports = {
       "exports": "always-multiline",
       "functions": "ignore"
     }],
-    "import/no-extraneous-dependencies": [2, {
-      "devDependencies": true
-    }],
+    "import/no-extraneous-dependencies": "off",
     "no-unused-vars": "off",
```

Comment:
🤔 and I don't why this is off, it should be on too

Reply:
Are this and [that](https://github.com/nervosnetwork/neuron/pull/1802#discussion_r474162544) the same thing?